src/components/Profile.jsx

```diff
@@ -16,5 +17,15 @@ class Profile extends React.Component {
 		);
 	}
 }
+Profile.propTypes = {
+	user : PropTypes.shape({
+		images: PropTypes.arrayOf(PropTypes.shape({
+			small: PropTypes.string,
+			large: PropTypes.string,
+		})),
+		firstName: PropTypes.string,
+		lastName: PropTypes.string,
+	}),
```

Comment:
We access `this.props.user`'s keys without checking if it's provided, so it should be isRequired.